### PR TITLE
Bootstrap keyspaces controller

### PR DESCRIPTION
**Pull request by vishalkatikineni**
Issue 1801

Description of changes: Bootstrapping keyspaces controller

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ATTRIBUTION.md
Comment (a-hilaly) [inline ATTRIBUTION.md:1]: Can you copy the content of https://github.com/aws-controllers-k8s/dynamodb-controller/blob/main/ATTRIBUTION.md here

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners#owners_aliases
+
+aliases:
+  core-ack-team:
+    - a-hilaly
+    - RedbackThomson
+    - jljaco
```
Comment (a-hilaly) [inline OWNERS_ALIASES:7]: Please use:
```
aliases:
  core-ack-team:
    - a-hilaly
    - jlbutler
    - ivelichkovich
    - jljaco
  emeritus-core-ack-team:
    - jaypipes
    - RedbackThomson
    - vijtrip2
```

File: apis/v1alpha1/ack-generate-metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+ack_generate_info:
+  build_date: "2023-12-21T21:47:08Z"
+  build_hash: a4bd77b1cacbbbfa7aec7325e5e93dd92d64394b
+  go_version: go1.20.4
+  version: v0.26.0-4-ga4bd77b
```
Comment (a-hilaly) [inline apis/v1alpha1/ack-generate-metadata.yaml:5]: Can we generate the controller using `code-generator` v0.28.0?

File: metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+service:
+  full_name:
+  short_name:
+  link:
+  documentation:
```
Comment (a-hilaly) [inline metadata.yaml:5]: Can we fill those with working URLs? e.g https://github.com/aws-controllers-k8s/dynamodb-controller/blob/main/metadata.yaml

**Review by a-hilaly — COMMENTED** [review]
Thank you @vishalkatikineni - i have few comments in-line

File: ATTRIBUTION.md
Comment (vishalkatikineni) [reply ATTRIBUTION.md:1]: Updated ATTRIBUTION.md

File: OWNERS_ALIASES (hunk 1 above)
Comment (vishalkatikineni) [reply OWNERS_ALIASES]: Updated OWNER_ALIASES

File: apis/v1alpha1/ack-generate-metadata.yaml (hunk 2 above)
Comment (vishalkatikineni) [reply apis/v1alpha1/ack-generate-metadata.yaml]: re-generated using `code-generator` v0.28.0

File: metadata.yaml (hunk 3 above)
Comment (vishalkatikineni) [reply metadata.yaml]: updated metadata.yaml

**Review by a-hilaly — APPROVED** [review]
Thank you @vishalkatikineni !

**Pull request merged**